packages/ui/src/app/dashboard/clientPage.tsx

```diff
@@ -1,16 +1,5 @@
 'use client';
 
-import {
```

Comment:
Same here, make sure we install the icons package within this package so we don't get any surprises